nbs/examples/Forecasting_TFT.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>I think you already mentioned it. Repetion feels probably necessary just because we spend a lot of time with wrangling. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>